codonPython/file_utils.py

```diff
@@ -0,0 +1,312 @@
+import pandas as pd
+import os
+
+def file_search(path = '.', doctype = 'csv', like = [''], strict = False):
+    """
+    This function creates a list of all files of a certain type, satisfying the criteria outlined
+    in like = [...] parameter. The function only searches for files in the specified folder
+    of the current working directory that is set by the user.
+    
+    Parameters
+    -----------
+    path : string 
+        Path to a folder in the current working directory 
+        default = '.', i.e. current working directory folder
+    doctype : string
+        Document format to search for 
+        e.g. 'csv' or 'xlsx'
+        default = 'csv'
+    like : list
+        A list of words to filter the file search on 
+        default = [''], i.e. no filter
+    strict : bool
+        Set True to search for filenames containing all words from 'like' list (
+        default = False
+        
+    Returns
+    -------
+    list
+    
+    Examples
+    -------
+    '>>> file_search()'

```

Comment:
Remove the quotation marks gere and the examples will be run as part of the tests